I just can't get flow-typed to fail test

No mater what I write in my test file even if it's just `hello there` I still get this:

```
flow-typed run-tests
Running definition tests in /Users/gozala/Projects/flow-typed/definitions/npm...

Fetching all Flow binaries...

All tests passed!
```

Oddly enough I also can't get `./travis.sh` to succeed on my machine here is the output I get:

```
yarn install v1.1.0
warning You are using Node "9.3.0" which is not supported and may encounter bugs or unexpected behavior. Yarn supports the following semver range: "^4.8.0 || ^5.7.0 || ^6.2.2 || ^8.0.0"
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.64s.
yarn run v1.1.0
warning You are using Node "9.3.0" which is not supported and may encounter bugs or unexpected behavior. Yarn supports the following semver range: "^4.8.0 || ^5.7.0 || ^6.2.2 || ^8.0.0"
$ eslint **/**/test_*

/Users/gozala/Projects/flow-typed/definitions/npm/backbone-collection_v1.x.x/test_backbone-collection_v1.x.x.js
   5:14  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
   8:14  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  11:22  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  30:29  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  33:23  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  35:17  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  36:16  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/backbone-model_v1.x.x/test_backbone-model_v1.x.x.js
   6:14  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
   9:14  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  12:22  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  36:52  warning  Unexpected use of weak type "Object"    flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/backbone_v1.x.x/test_backbone-v1.x.x.js
  13:14  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  15:14  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  17:22  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  47:29  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  50:23  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  52:17  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  53:16  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/blacklist_v1.x.x/test_blacklist_v1.x.x.js
  4:33  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  5:37  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/bunyan_v1.x.x/test_bunyan_v1.x.x.js
  17:20  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/camelcase-keys_v3.x.x/test_camelcase-keys_v3.x.x.js
  3:15  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/commander_v2.x.x/test_commander_v2.x.x.js
  37:30  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/deepmerge_v1.x.x/test_deepmerge_v1.x.x.js
   7:16  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  31:37  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  31:48  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/express_v4.16.x/test_express_v4.16.x.js
  84:23  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/express_v4.x.x/test_express_v4.x.x.js
  69:23  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/fuzzaldrin_v2.x.x/test_fuzzaldrin_v2.js
  6:10  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/idb-keyval_v2.x.x/test_idb-keyval-v2.x.x.js
  16:44  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/idb_v2.x.x/test_idb-v2.x.x.js
  30:37  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/inline-style-prefixer_v3.x.x/test_inline-style-prefixer_v3.x.x.js
  5:15  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/inquirer_v2.x.x/test_rx_observable_array_v2.x.x.js
  41:17  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/inquirer_v2.x.x/test_rx_observable_create_v2.x.x.js
  9:9  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/isomorphic-fetch_v2.x.x/test_isomorphic-fetch_v2.js
  7:25  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/klasa_v0.3.x/test_klasa_v0.3.js
  17:22  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  19:18  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  49:24  warning  Unexpected use of weak type "any"     flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/koa-bunyan-logger_v2.x.x/test_koa_bunyan_logger_v2.x.x.js
  19:20  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/map-obj_v1.x.x/test_map-obj_v1.x.x.js
  3:18  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  3:79  warning  Unexpected use of weak type "any"     flowtype/no-weak-types
  3:92  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/mocha_v2.4.x/test_mocha_v2.4.x_hooks.js
   8:16  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  19:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  31:15  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  42:19  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/mocha_v2.4.x/test_mocha_v2.4.x_it.js
   8:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  26:25  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  46:25  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/mocha_v3.1.x/test_mocha_v3.1.x_hooks.js
   8:16  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  19:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  31:15  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  42:19  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/mocha_v3.1.x/test_mocha_v3.1.x_it.js
   9:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  25:25  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  43:25  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/mocha_v4.x.x/test_mocha_v4.x.x_hooks.js
   8:16  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  19:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  31:15  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  42:19  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/mocha_v4.x.x/test_mocha_v4.x.x_it.js
   9:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  25:25  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  43:25  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/morgan_v1.x.x/test_morgan.js
  29:58  warning  Unexpected use of weak type "any"  flowtype/no-weak-types
  29:68  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/normalizr_v2.x.x/test_normalizr-v2.js
  4:40  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  5:42  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  8:25  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/object-assign_v4.x.x/test_object-assign_v4.x.x.js
  5:16  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  6:16  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/papaparse_v4.x.x/test_papaparse_v4.x.x.js
  28:17  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  31:25  warning  Unexpected use of weak type "any"     flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/pify_v3.x.x/test_pify_v3.x.x.js
  3:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/primus_v5.x.x/test_primus-v5.js
  9:63  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/primus_v6.x.x/test_primus-v6.js
  9:63  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/prismic.io_v3.x.x/test_prismic.io_v3.x.x.js
   44:37  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   49:44  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   53:44  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   55:72  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   56:59  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   71:27  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   74:43  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  103:29  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  140:35  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  148:43  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  157:37  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  164:28  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  165:34  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  166:45  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  302:29  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/quill-delta_v3.5.x/test_quill-delta_v3.5.x.js
    7:21  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   10:16  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
   97:15  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  105:15  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  113:38  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  127:37  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  133:37  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/react-i18next_v6.x.x/test_react-i18next_v6.x.x.js
  101:37  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/react-jss_v7.x.x/test_react-jss_v7.x.x.js
  161:12  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/react-select_v1.x.x/test_react-select_v1.x.x.js
   7:31  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  10:31  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  11:30  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  12:33  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  13:32  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  19:32  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  20:31  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  63:21  warning  Unexpected use of weak type "any"     flowtype/no-weak-types
  67:26  warning  Unexpected use of weak type "any"     flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/redux-devtools-extension_v2.x.x/test_redux-devtools-extension_v2.x.x.js
  69:21  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  70:26  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/redux-form_v5.x.x/test_redux-form_v5.js
  19:25  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  20:20  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  21:21  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
  33:32  warning  Unexpected use of weak type "any"       flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/redux-saga-thunk_v0.5.x/test_redux-saga-thunk_v0.5.x.js
  10:43  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/rxjs_v5.0.x/test_rxjs.js
   92:34  warning  Unexpected use of weak type "any"  flowtype/no-weak-types
  103:19  warning  Unexpected use of weak type "any"  flowtype/no-weak-types
  217:56  warning  Unexpected use of weak type "any"  flowtype/no-weak-types
  218:52  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/sinon_v2.x.x/test_sinon_v2.x.x.js
    4:19  warning  Unexpected use of weak type "Function"  flowtype/no-weak-types
    6:20  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
   57:22  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
   58:18  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  166:16  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  175:16  warning  Unexpected use of weak type "any"       flowtype/no-weak-types
  185:16  warning  Unexpected use of weak type "any"       flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/twit_v2.x.x/test_twit_v2.x.x.js
  16:30  warning  Unexpected use of weak type "any"  flowtype/no-weak-types
  17:29  warning  Unexpected use of weak type "any"  flowtype/no-weak-types

/Users/gozala/Projects/flow-typed/definitions/npm/underscore_v1.x.x/test_underscore-v1.js
  171:20  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types
  172:13  warning  Unexpected use of weak type "Object"  flowtype/no-weak-types

✖ 130 problems (0 errors, 130 warnings)

✨  Done in 9.34s.
yarn install v1.1.0
warning You are using Node "9.3.0" which is not supported and may encounter bugs or unexpected behavior. Yarn supports the following semver range: "^4.8.0 || ^5.7.0 || ^6.2.2 || ^8.0.0"
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
$ mkdirp dist && npm run test

> flow-typed@2.2.3 test /Users/gozala/Projects/flow-typed/cli
> npm run clean && npm run build && npm run test-quick


> flow-typed@2.2.3 clean /Users/gozala/Projects/flow-typed/cli
> rimraf dist


> flow-typed@2.2.3 build /Users/gozala/Projects/flow-typed/cli
> mkdirp dist && babel ./src --out-dir=./dist

src/cli.js -> dist/cli.js
src/commands/__tests__/install-test.js -> dist/commands/__tests__/install-test.js
src/commands/__tests__/runTests-test.js -> dist/commands/__tests__/runTests-test.js
src/commands/__tests__/search-test.js -> dist/commands/__tests__/search-test.js
src/commands/create-stub.js -> dist/commands/create-stub.js
src/commands/install.js -> dist/commands/install.js
src/commands/runTests.js -> dist/commands/runTests.js
src/commands/search.js -> dist/commands/search.js
src/commands/update-cache.js -> dist/commands/update-cache.js
src/commands/update.js -> dist/commands/update.js
src/commands/validateDefs.js -> dist/commands/validateDefs.js
src/commands/version.js -> dist/commands/version.js
src/lib/TEST_UTILS.js -> dist/lib/TEST_UTILS.js
src/lib/__mocks__/node.js -> dist/lib/__mocks__/node.js
src/lib/__tests__/cacheRepoUtils-test.js -> dist/lib/__tests__/cacheRepoUtils-test.js
src/lib/__tests__/codeSign-test.js -> dist/lib/__tests__/codeSign-test.js
src/lib/__tests__/flowVersion-test.js -> dist/lib/__tests__/flowVersion-test.js
src/lib/__tests__/libDefs-test.js -> dist/lib/__tests__/libDefs-test.js
src/lib/__tests__/semver-test.js -> dist/lib/__tests__/semver-test.js
src/lib/cacheRepoUtils.js -> dist/lib/cacheRepoUtils.js
src/lib/codeSign.js -> dist/lib/codeSign.js
src/lib/fileUtils.js -> dist/lib/fileUtils.js
src/lib/flowProjectUtils.js -> dist/lib/flowProjectUtils.js
src/lib/flowVersion.js -> dist/lib/flowVersion.js
src/lib/git.js -> dist/lib/git.js
src/lib/github.js -> dist/lib/github.js
src/lib/isInFlowTypedRepo.js -> dist/lib/isInFlowTypedRepo.js
src/lib/libDefs.js -> dist/lib/libDefs.js
src/lib/node.js -> dist/lib/node.js
src/lib/npm/__tests__/npmLibDefs-test.js -> dist/lib/npm/__tests__/npmLibDefs-test.js
src/lib/npm/npmLibDefs.js -> dist/lib/npm/npmLibDefs.js
src/lib/npm/npmProjectUtils.js -> dist/lib/npm/npmProjectUtils.js
src/lib/semver.js -> dist/lib/semver.js
src/lib/stubUtils.js -> dist/lib/stubUtils.js
src/lib/validationErrors.js -> dist/lib/validationErrors.js

> flow-typed@2.2.3 test-quick /Users/gozala/Projects/flow-typed/cli
> jest && npm run lint

 PASS  src/lib/npm/__tests__/npmLibDefs-test.js
  ● Console

    console.log src/lib/cacheRepoUtils.js:128
      • rebasing flow-typed cache...

 PASS  src/commands/__tests__/install-test.js
 PASS  src/lib/__tests__/cacheRepoUtils-test.js
 PASS  src/lib/__tests__/libDefs-test.js
 PASS  src/commands/__tests__/search-test.js
 PASS  src/lib/__tests__/flowVersion-test.js
 PASS  src/lib/__tests__/codeSign-test.js
 PASS  src/lib/__tests__/semver-test.js
 FAIL  src/commands/__tests__/runTests-test.js (7.238s)
  ● run-tests (command) › regression 1385 › returns error code if $ExpectError tes
t doesn't fail

    expect(received).toEqual(expected)

    Expected value to equal:
      1
    Received:
      0

      at _callee2$ (src/commands/__tests__/runTests-test.js:23:30)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:64:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:355:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:116:21)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:64:40)
      at invoke (node_modules/regenerator-runtime/runtime.js:154:20)
      at node_modules/regenerator-runtime/runtime.js:201:11
          at new Promise (<anonymous>)
      at callInvokeWithMethodAndArg (node_modules/regenerator-runtime/runtime.js:200:16)
      at AsyncIterator.enqueue [as _invoke] (node_modules/regenerator-runtime/runtime.js:223:13)
      at AsyncIterator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:116:21)
      at Object.<anonymous>.runtime.async (node_modules/regenerator-runtime/runtime.js:244:14)
      at Object._callee2 (src/commands/__tests__/runTests-test.js:22:111)
          at new Promise (<anonymous>)
      at Promise.resolve.then.el (node_modules/p-map/index.js:42:16)
          at <anonymous>

  ● run-tests (command) › regression 1385 › console logs about unused suppression

    expect(collection)[.not].toContainEqual(value)

    Expected collection to be an array-like structure.
    Received: undefined

      at _callee3$ (src/commands/__tests__/runTests-test.js:32:36)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:64:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:355:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:116:21)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:64:40)
      at invoke (node_modules/regenerator-runtime/runtime.js:154:20)
      at node_modules/regenerator-runtime/runtime.js:201:11
          at new Promise (<anonymous>)
      at callInvokeWithMethodAndArg (node_modules/regenerator-runtime/runtime.js:200:16)
      at AsyncIterator.enqueue [as _invoke] (node_modules/regenerator-runtime/runtime.js:223:13)
      at AsyncIterator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:116:21)
      at Object.<anonymous>.runtime.async (node_modules/regenerator-runtime/runtime.js:244:14)
      at Object._callee3 (src/commands/__tests__/runTests-test.js:26:135)
          at new Promise (<anonymous>)
      at Promise.resolve.then.el (node_modules/p-map/index.js:42:16)
          at <anonymous>

Test Suites: 1 failed, 8 passed, 9 total
Tests:       2 failed, 96 passed, 98 total
Snapshots:   0 total
Time:        9.859s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! flow-typed@2.2.3 test-quick: `jest && npm run lint`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the flow-typed@2.2.3 test-quick script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gozala/.npm/_logs/2018-01-11T22_43_45_200Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! flow-typed@2.2.3 test: `npm run clean && npm run build && npm run test-quick`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the flow-typed@2.2.3 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gozala/.npm/_logs/2018-01-11T22_43_45_291Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

--- a/cli/src/commands/runTests.js
+++ b/cli/src/commands/runTests.js
@@ -103,7 +103,7 @@ async function getOrderedFlowBinVersions(
       const GH_CLIENT = gitHubClient();
       // We only test against the latest numberOfReleases Versions
       const QUERY_PAGE_SIZE = numberOfReleases;
-      const OS_ARCH_FILTER_RE = new RegExp(BIN_PLATFORM);
+      const OS_ARCH_FILTER_RE = new RegExp(`flow-${BIN_PLATFORM}`);
 
       let binURLs = new Map();
       let apiPayload = null;

